static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -35,6 +39,29 @@ define([
                     bean.on(el, 'click', askQuestion);
                 });
             }
+
+            var storyQuestionsComponent = document.querySelector('.js-view-tracking-component');
+    
+            mediator.on('window:throttledScroll', function onScroll() {
+                var height = detect.getViewport().height;
+                var coords = storyQuestionsComponent.getBoundingClientRect();
+                var isStoryQuestionsInView = coords.top + window.scrollY > 0 && coords.bottom < height;
+                
+                if( isStoryQuestionsInView ) {
+                    var atomId = $('.js-storyquestion-atom-id').attr('id');
```

Comment:
perhaps you can drop the `SPAN` and move the id to the surrounding `DIV`, which you already have as `storyQuestionsComponent`